cms/envs/common.py

```diff
@@ -1008,6 +1008,7 @@
     XModuleMixin,
     EditInfoMixin,
     AuthoringMixin,
+    'cms.lib.xblock.upstream_sync.UpstreamSyncMixin',
```

Comment:
Kinda. It was that we can't load any djangoapps from settings files, and `lib/upstream_sync.py` imported from a couple djangoapps.

In my latest commit, I went back and made this a proper import, and then changed move some upstream_sync.py imports into their functions to avoid the problem, each with an explanatory comment.